src/components/radios/_radios.scss

```diff
@@ -10,6 +10,7 @@
 @include govuk-exports("govuk/component/radios") {
   $govuk-radios-size: govuk-spacing(7);
   $govuk-radios-label-padding-left-right: govuk-spacing(3);
+  $govuk-radios-focus-width: 4px;
```

Comment:
Assuming we _need_ to add 1 pixel can this be `$govuk-focus-width + 1` so that we understand where this value came from?